falco/templates/daemonset.yaml

```diff
@@ -64,10 +64,12 @@ spec:
             - --cri
             - /run/containerd/containerd.sock
             {{- end }}
+            {{- if .Values.kubernetesSupport.enabled }} 
             - -K
-            - /var/run/secrets/kubernetes.io/serviceaccount/token
+            - {{ .Values.kubernetesSupport.certFile }} 
             - -k
-            - "https://$(KUBERNETES_SERVICE_HOST)"
+            - {{ .Values.kubernetesSupport.apiUrl }} 
+            {{- end }}
             - -pk
```

Comment:
maybe `-pk` should have been _inside_ the `if` block?

Reply:
`-pk` is just about the output format. I believe we can leave it *outside* the if block (as you did), so we will not break output consumers who rely on that format.